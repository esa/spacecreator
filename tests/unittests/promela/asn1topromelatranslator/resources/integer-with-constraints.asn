Ints DEFINITIONS ::= BEGIN

ConstraintedInt ::= INTEGER(0 .. 10)

MyInt ::= ConstraintedInt(2 .. 5)

END