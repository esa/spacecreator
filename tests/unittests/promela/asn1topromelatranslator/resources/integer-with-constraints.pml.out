inline MyInt_generate_value(MyInt_gv)
{
    int MyInt_tmp;
    if
    ::  atomic {
        select (MyInt_tmp : 2 .. 5);
    }
    fi;
    MyInt_gv = MyInt_tmp;
}
